src/Windows/R/Components/Impl/InteractiveWorkflow/Commands/SessionInformationCommand.cs

```diff
@@ -88,16 +88,21 @@ public sealed class SessionInformationCommand : IAsyncCommand {
             sb.AppendLine("\t" + Resources.PhysicalMemory.FormatInvariant(aboutHost.TotalPhysicalMemory, aboutHost.FreePhysicalMemory));
             sb.AppendLine("\t" + Resources.VirtualMemory.FormatInvariant(aboutHost.TotalVirtualMemory, aboutHost.FreeVirtualMemory));
 
-            if (!string.IsNullOrEmpty(aboutHost.VideoCardName)) {
-                sb.AppendLine("\t" + Resources.VideoCardName.FormatInvariant(aboutHost.VideoCardName));
-            }
+            int count = 1;
+            foreach(var cardInfo in aboutHost.VideoCards) {
```

Comment:
yes. WSL for example does not have display device. 'lshw -c display' will return nothing.

---
In reply to: [116114194](https://github.com/Microsoft/RTVS/pull/3516#discussion_r116114194) [](ancestors = 116114194)